setup.py

```diff
@@ -18,5 +18,8 @@
           "Programming Language :: Python :: 2",
           "Programming Language :: Python :: 3",
           "Topic :: Scientific/Engineering",
+      ],
+      data_files=[
```

Comment:
And the last thing related to conventions: We can just use the script from the repository itself, and not install it.
Maybe we can add a packaged CLI tool in the future that will implement this functionality as a subcommand, but this requires a bit more thought.